packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -1230,6 +1246,7 @@ export function useGlobalStylesOutputWithConfig( mergedConfig = {} ) {
 			disableLayoutStyles,
 			isTemplate
 		);
+		console.log( 'globalStyles', globalStyles );
```

Comment:
This is a marker to remind me that `getThemeFileURI` isn't implemented for global styles yet either.

Maybe it needs to be done by injecting a CSS rule into the DOM post-render, something like layout does.

I don't know. This is getting out of hand. 😄